app/assets/javascripts/select-kit/addon/components/select-kit.js

```diff
@@ -866,6 +879,20 @@ export default Component.extend(
 
                   wrapper.style.width = `${this.element.offsetWidth}px`;
                   wrapper.style.height = `${height}px`;
+                  if (placementStrategy === "fixed") {
```

Comment:
The tl;dr is this fixes this problem when select-kit is used in a modal:

![image](https://user-images.githubusercontent.com/17474474/106000891-5fab9600-60c0-11eb-8a10-958346ad2a92.png)

----

Select-kit has a `.select-kit-body` element which contains the options that you can select from. The body is positioned by popper.js which takes a `placementStrategy` param that can be either `fixed` or `absolute`. When select-kit is inside a modal, the placement strategy is set to `fixed`, otherwise it's set to `absolute`. But select-kit has another element `.select-kit-wrapper` which gives select-kit the glowing effect. The wrapper's CSS `position` should be the same as the placement strategy of the body element, but it's currently always `absolute` which is what is causing the glowing effect to be cut off in the screenshot above. What I'm doing here is I added a check for `placementStrategy` and if it's `fixed`, then the wrapper's `position is updated ` to `fixed` and the `left` and `top` values are set accordingly.

I apologize if I over-explained things, but I wanted to mention the full story in case you have a better approach to fix this problem.